insolar/assured-ledger
# PLAT-280: SM for handle VDelegateRequestFinished

**- What I did**
Added new SMVDelegatedRequestFinished for handle message. Also, added step to access shared data with object sm. update state and decrement pending counters. Release sync object for pending ordered operation if pending counters are 0.

**- How I did it**
 Write a code)
**- How to verify it**
virtual/integration/vdelegatedrequestfinished_test.go for 
Check that execute call will be called only after VDelegatedRequestFinished if we have pending ordered operation in VStateReport
**- Description for the changelog**

https://insolar.atlassian.net/browse/PLAT-280
